docs/manual/java/guide/build/index.toc

```diff
@@ -1,2 +1,8 @@
 LagomBuild:Defining a Lagom build
-MultipleBuilds:Splitting a system into multiple builds
\ No newline at end of file
+MultipleBuilds:Splitting a system into multiple builds
```

Comment:
Should we use the new multiple "Next" links feature here? Right now, the "Next" link from "Splitting a system into multiple builds" goes straight to "Importing an existing Maven project into Eclipse".